packages/ts-config/src/tsconfig.json

```diff
@@ -8,7 +8,7 @@
 		"esModuleInterop": true,
 		"importHelpers": false,
 		"incremental": true,
-		"lib": ["esnext"],
+		"lib": ["ESNext", "DOM"],
```

Comment:
God to the no, if you want to add the types, add it just to that one file, not globally across all projects.